plugins/@grouparoo/cloudwatch/src/initializers/plugin.ts

```diff
@@ -35,9 +35,9 @@ export default class CloudwatchInitializer extends Initializer {
       },
       jsonMessage: true,
       retentionInDays: 7, // store the logs for 7 days
-      awsAccessKeyId: process.env.S3_ACCESS_KEY,
-      awsSecretKey: process.env.S3_SECRET_ACCESS_KEY,
-      awsRegion: process.env.S3_REGION,
+      awsAccessKeyId: process.env.AWS_ACCESS_KEY,
```

Comment:
Can we use `AWS_ACCESS_KEY_ID` instead of `AWS_ACCESS_KEY`? When setting it up I was a bit confused what exactly it was. I've seen `AWS_ACCESS_KEY_ID` used more often by other tools and is even auto loaded by the sdk https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/loading-node-credentials-environment.html

Reply:
You got it!